index.js

```diff
@@ -18,10 +20,11 @@ var push = [].push
  *
  * @param {EstreeNode} tree
  * @param {EstreeComment[]} [comments]
+ * @param {boolean} isEstreeNode
  */
-export function attachComments(tree, comments) {
+export function attachComments(tree, comments, isEstreeNode = false) {
```

Comment:
As mentioned in https://github.com/wooorm/estree-util-attach-comments/issues/5#issuecomment-826370607, what this toggles is not “estree node” style, but rather: espree style vs. recast style.
Could you rename the option, and update the description?

Might also be better to put this in an options object.
And the `= false` part is unneeded, it’s `undefined` by default, which is in my opinion fine!

Could be `style: 'recast' | 'espree'` maybe? 🤔